lib/src/Uncertainty/Algorithm/Simulation/EventSimulation.cxx

```diff
@@ -226,7 +226,8 @@ Graph EventSimulation::drawProbabilityConvergence(const Scalar level) const
       dataUpperBound.add(pt);
     }
   }
-  const Curve estimateCurve(dataEstimate, "red", "solid", 2, "probability estimate");
+  Curve estimateCurve(dataEstimate, "probability estimate");
```

Comment:
This code is ok, but the code below is not: ` const Curve lowerBoundCurve(dataLowerBound, "green", "solid", 1, "bounds");`. The same color "green" is used for the lower and upper bounds. 

![image](https://github.com/openturns/openturns/assets/31351465/b3f66d57-94f5-4e5a-b01d-aa5a5f885a84)

Instead, the second default color of the palette should be used.